waterlog/src/Stylesheets/_wastage.scss

```diff
@@ -7,7 +7,19 @@
 	color: #4F5B62;
 	size: 14pt;
 }
+.wastageTank{
 
+	background-color:#4F5B62;
+  margin: 20px;
```

Comment:
Inconsistent indentations and add line between declarations

Reply:
Done